vars/gangplank.groovy

```diff
@@ -0,0 +1,87 @@
+// Run gangplank
+def ganplankCmd
```

Comment:
Hmm, seems like we could pretty easily get rid of this global var by having `getMode()` return it and then passing it in to the various helpers and then finally `runGangplank()`.